examples/dev-server-fargate/main.tf

```diff
@@ -47,16 +47,27 @@ resource "aws_ecs_service" "example_client_app" {
 }
 
 module "example_client_app" {
-  source = "../../modules/mesh-task"
-  family = "${var.name}-example-client-app"
-  port   = "9090"
+  source           = "../../modules/mesh-task"
+  consul_ecs_image = "hashicorpdev/consul-ecs:d1afd83"
+  family           = "${var.name}-example-client-app"
+  port             = "9090"
   upstreams = [
     {
       destination_name = "${var.name}-example-server-app"
       local_bind_port  = 1234
     }
   ]
   log_configuration = local.example_client_app_log_config
+  checks = [
+    {
+      CheckID  = "api"
+      Name     = "HTTP API on port 9090"
+      HTTP     = "http://localhost:9090/health"
+      Method   = "GET"
+      Interval = "10s"
+      Timeout  = "1s"
+    }
+  ]
```

Comment:
Hmm, I guess longer term we'd recommend people to use the (future) "health sync" approach, instead of defining Consul native checks. So I think it makes sense to remove this.